types/src/on_chain_config/consensus_config.rs

```diff
@@ -128,8 +128,10 @@ pub enum ProposerElectionType {
     // Round robin rotation of proposers
     // with specified param contiguous_rounds
     RotatingProposer(u32),
-    // Committed history based proposer election
+    // Committed history based proposer election, with wrong window order
     LeaderReputation(LeaderReputationType),
+    // Committed history based proposer election, with correct window order
+    LeaderReputationV2(LeaderReputationType),
```

Comment:
@igor-aptos - We need to add this to the end otherwise, it is going to mess up the enum ordering and is not backward compatible.